Update Home Page Content (#7)
dont-rely-on-nulls/dont-rely-on-nulls.github.io: content/english/_index.md

banner:
  title: "Don't rely on Nulls, rely on us"
  content: 

>     "Fear of unreliable software? Rest and allow us to make your dream a reality with our unmatched attention to detail and diligence towards reputable software"

Reviewer: The "rest" bit feels off. It really doesn't sound like something a native speaker would say. Maybe "rest easy" or "relax" instead?

>     "Fear of unreliable software? Rest and allow us to make your dream a reality with our unmatched attention to detail and diligence towards reputable software"

Reviewer: ```suggestion
    "Fear of unreliable software? Rest and allow us to make your dream a reality with our unmatched attention to detail and diligence towards reputable software."
```

Author: A subtitle ending with dot feels a little bit off.

# Features
features:
  - title: "Quality as Paramount"
    image: "/images/service-1.png"

>     content: "In our projects, we merge practical expertise in functional programming with a commitment to delivering solutions on time and within budget --- you can trust us that your product will fly in no time. We focus on efficient delivery at low cost, without compromising on quality and transparency. Our team of dedicated professionals leverages both practical and abstract strategies to create reliable software solutions that meet our clients' requirements. We value partnerships with the open-source community, fostering an environment where diverse perspectives converge to shape tailored solutions that best fit the product."

Reviewer: The "fly in no time" bit sounds redundant, after we just said we were committed to delivering solutions on time. I'm not quite sure how to change it, though. A good candidate is "soar", but that sounds slightly pretentious in this context. I'm not sure that we would want that while simultaneously claiming to be humble. Also good is "shine", but that has no connotation of height.

Reviewer: Perhaps "will soon fly"?

>     content: "In our projects, we merge practical expertise in functional programming with a commitment to delivering solutions on time and within budget --- you can trust us that your product will fly in no time. As a boutique, we focus on efficient delivery at low cost, without compromising on quality and transparency. Our team of dedicated professionals leverages both practical and abstract strategies to create reliable software solutions that meet our clients' requirements.  We value partnerships with the open-source community, fostering an environment where diverse perspectives converge to shape tailored solutions that best fit the product."

Reviewer 2: The `as a boutique we <something>` feels off. It begs for a justification of why that is the case as a boutique. I would just remove it or change `boutique` for something else

    button:
      enable: true
      label: "Get to know Us"
      link: "team"

  - title: "Humility and Learning"
    image: "/images/service-2.png"

>     content: "Integrity is paramount to our approach. We adhere to the highest ethical standards and craftsmanship to ensure our solutions align seamlessly with our clients' objectives. In a rapidly evolving tech landscape, we remain humble learners, valuing grounded ideas and the pursuit of improvement. Our team embraces ongoing education and exploration to shape the future of software development. Collaboration is at the heart of our ethos. Our hobby projects demonstrate how dedicated we are into exploration and facing uncharged territories."

Reviewer: ```suggestion
    content: "Integrity is key to our approach. We adhere to the highest ethical standards and craftsmanship to ensure our solutions align seamlessly with our clients' objectives. In a rapidly evolving tech landscape, we remain humble learners, valuing grounded ideas and the pursuit of improvement. Our team embraces ongoing education and exploration to shape the future of software development. Collaboration is at the heart of our ethos. Our hobby projects demonstrate how dedicated we are to braving uncharted territories."
```

Brevity is wit. Also, it should be "uncharted".

    button:
      enable: true
      label: "Check Projects"
      link: "projects"

  - title: "Business Value via FP"
    image: "/images/service-3.png"

>     content: "Choose Don't rely on Nulls for dependable software solutions. Join us on a journey where practical innovation meets from-first-principles reflection, guided by our expertise in functional programming, towards a brighter digital horizon. The outcome will be pretty, and maintanable enough that you will like to frame it."

Reviewer: ```suggestion
    content: "Choose Don't rely on Nulls for dependable software solutions. Join us on a journey where practical innovation meets reflection from first principles, guided by our expertise in functional programming, towards a brighter digital horizon. The outcome will be pretty, and maintanable enough that you'd like to frame it."
```

The "will" there feels rather forceful, more like a command than a statement of fact. You can drop the contraction and say "would", but that doesn't read as smoothly.

I don't see the point of a hyphenated adjective here either. The normal way of saying it works just fine.
